docs/config.md

```diff
@@ -777,6 +777,7 @@ or the root domain:
     ```
     # Note that this config is most certainly incomplete. Please help out and let me know what's missing
     # via Discord/Matrix or in a GitHub issue.
+    # Note: Caddy automatically handles both HTTP and WebSockets with reverse_proxy 
```

Comment:
Upon request I can supply a docker compose file with caddy and ntfy. But currently I am not sure if this adds any value (looking for confirmation) 

- adding a docker compose for caddy can get stale
- caddy file will display caddy docker integration (which may not provide any value for the documentation of this project)
  - below is a sample where ntfy is the docker container name and 80 is the default ntfy docker port (not machine port but docker container port)

````
ntfy.sh, http://nfty.sh {
      reverse_proxy ntfy:80
}
````